src/test/java/org/gridsuite/ds/server/controller/DynamicSimulationControllerIEEE14Test.java

```diff
@@ -232,9 +238,19 @@ public void test01GivenCurvesAndEvents() throws Exception {
         String jsonResultTimeSeries = TimeSeries.toJson(resultTimeSeries);
 
         // export result to file
-        FileUtils.writeStringToFile(this, outputDir + RESOURCE_PATH_DELIMITER + "exported_" + RESULT_SIM_JSON, jsonResultTimeSeries);
+        FileUtils.writeBytesToFile(this, outputDir + RESOURCE_PATH_DELIMITER + "exported_" + RESULT_SIM_JSON, jsonResultTimeSeries.getBytes());
 
         // compare result only timeseries
         assertThat(objectMapper.readTree(jsonResultTimeSeries)).isEqualTo(objectMapper.readTree(jsonExpectedTimeSeries));
+
+        // check dump file not empty
+        byte[] outputState = dynamicSimulationResultService.getOutputState(runUuid);
+        assertThat(outputState).isNotEmpty();
```

Comment:
```suggestion
       assertThat(outputState)
    .withFailMessage("Output state for the dynamo simulation is empty ...")
    .isNotEmpty();
```

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/116/commits/85a0baaed9feae2f53cfbfbf62f944076e27df09